dbt/models/qc/schema.yml

```diff
@@ -486,3 +486,124 @@ models:
           - dbt_utils.accepted_range:
               name: qc_vw_neg_asmt_value_valasm3_gte_0
               <<: *test-non-negative
+              
+  - name: qc.vw_iasworld_sales_null_values
+    description: '{{ doc("view_vw_iasworld_sales_null_values") }}'
+    config:
+      tags:
+        - test_qc_sales
+    tests:
+      - expression_is_true:
+          name: qc_vw_iasworld_sales_null_values_deed_null_count_lte_1000
+          expression: deed_null_count < 1000
```

Comment:
**[Question, blocking]** What's the significance of 1,000 here? Is it just that we know there are already a couple hundred pre-existing null rows? I'm curious in particular if these null rows are expected and hence do not represent bad data, or if instead they're data errors that we're already aware of; if it's the latter, I think we should probably remove the restriction so that we can track the full extent of missing data.

Reply:
@dfsnow might have thoughts on this as well, but there are often missing values which could be seen as "acceptable". One of the issues that we ran into was that NULL values increased from say 25 to 15,000. Having a relatively high threshold means that things fail when data integrity changes, rather than "typical" data entry failures.

Reply:
Some amount of null values is indeed acceptable, but setting what that threshold should be is likely going to need further discussion. You might consider something like [dbt-utils.not_null_proportion](https://github.com/dbt-labs/dbt-utils#not_null_proportion-source) instead of a raw count threshold.